app.js

```diff
@@ -736,7 +749,7 @@ function getBoatPattern(startingRow = 8, startingColumn = 8) {
 }
 
 // The user can click this button to manually trigger a tick.
-const tickButton = document.querySelector('#tick');
+const tickButton = document.getElementById('tick');
```

Comment:
I think using getElementById, instead of querySelector, would be the better option all through the file. 
One, you don't have to write #'id'
Two, I think it's actually the better supported option